components/Footer.test.tsx

```diff
@@ -11,7 +11,7 @@ describe('Footer', () => {
   it('renders the footer links', () => {
     render(<Footer />);
     const footerLinks = screen.getAllByRole('link');
-    expect(footerLinks).toHaveLength(18);
+    expect(footerLinks).toHaveLength(14);
```

Comment:
Why did this change? we didn't change anything in the footer

Reply:
Ah I see, the changes below. This should go on a separate PR

Reply:
I changed as test was complaining about the usage of the same word in different fields